### PR TITLE
NPC Trial Dialog

### PR DESCRIPTION
**Pull request by allister-translator**
(´・ω・`)

### CODE REVIEW

File: Translated/Dialog/Dachua/trial_apc_7904001.csv [hunk 1]
```diff
@@ -0,0 +1,54 @@
+Begin#0,"""/ci1 1 Oh, well hello there. Think you could you lend me a hand here?"""
```
Comment (Frey8914) [inline Translated/Dialog/Dachua/trial_apc_7904001.csv]: /ci1 1 Oh, well hello there. Think you could lend me a hand here?
n.b. line is reused in same file

Comment (allister-translator) [reply Translated/Dialog/Dachua/trial_apc_7904001.csv]: yeah, multiple times. just sega being sega

File: Translated/Dialog/Dachua/trial_apc_7904001.csv [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Success#0,"""/ci2 1 ...Thanks for the help. Good luck out there."""
+Failure#0,"""/ci4 3 Things are looking bad! Sorry, I have to retreat."""
+Dead#0,"""/ci5 3 No... Not like this..."""
+Begin_v2#0,"""/ci1 1 Oh, well hello there. Think you could you lend me a hand here?"""
+Success_v2#0,"""/ci2 1 ...Thanks for the help. Good luck out there."""
+Failure_v2#0,"""/ci4 3 Things are looking bad! Sorry, I have to retreat."""
+Dead_v2#0,"""/ci5 3 No... Not like this..."""
+Begin_7002001#0,"""/ci1 2 Uh-oh. This isn't what I expected..."""
+Success_7002001#0,"""/ci2 1 Phew... Looks like we made it."""
+Failure_7002001#0,"""/ci4 3 Ngh... I think I might have broke something... I'm just gonna make my way back to the City, okay?"""
```
Comment (DabirSA) [inline Translated/Dialog/Dachua/trial_apc_7904001.csv:11]: "broken" something?

File: Translated/Dialog/Manon/trial_apc_7001001.csv [hunk 3]
```diff
@@ -52,9 +52,9 @@ Success_7002001_v3#2,"""＊＊成功(Ver3)"""
 Failure_7002001_v3#2,"""＊＊失敗(Ver3)"""
 Dead_7002001_v3#2,"""＊＊死亡(Ver3)"""
 BuddyDead_7002001_v3#2,"""＊＊相方死亡(Ver3)"""
-Begin#3,"""あっ、いいところに！<br>ちょっと数が多くて大変だから<br>手伝って貰えないかな？"""
-Success#3,"""ふう、なんとかなったぁ……<br>素直にありがとね、助かったよ。<br>それじゃ、またね！"""
-Failure#3,"""これ以上は危険よ！<br>ごめんなさい、先に撤退するわね。"""
+Begin#3,"""Oh, thank goodness! I'm a little in over my head here... Think you could give me a hand thinning out these numbers?"""
```
Comment (DabirSA) [inline Translated/Dialog/Manon/trial_apc_7001001.csv]: I'd go with "I'm in a little over my head" or "I'm in over my head a little".

File: Translated/Dialog/Dachua/trial_apc_7904001.csv (hunk 2 above)
Comment (allister-translator) [reply Translated/Dialog/Dachua/trial_apc_7904001.csv:11]: that line is aina speaking so i thought it fit her better. i can change it if you want

Comment (DabirSA) [reply Translated/Dialog/Dachua/trial_apc_7904001.csv:11]: Yeah it's probably fine.

File: Translated/Dialog/Dachua/trial_apc_7904001.csv (hunk 1 above)
Comment (allister-translator) [reply Translated/Dialog/Dachua/trial_apc_7904001.csv]: fixed, thank you for pointing this out

File: Translated/Dialog/Manon/trial_apc_7001001.csv (hunk 3 above)
Comment (allister-translator) [reply Translated/Dialog/Manon/trial_apc_7001001.csv]: agreed. fixed

**Pull request merged**